custom/icds_reports/utils.py

```diff
@@ -3936,12 +4055,12 @@ def get_data_for(filters):
         map_data.update({name: row_values})
 
     fills = OrderedDict()
-    fills.update({'Launched': GREEN})
+    fills.update({'Launched': PINK})
     fills.update({'defaultFill': GREY})
 
     return [
         {
-            "slug": "awc_daily_statuses",
+            "slug": "awcc_covered",
```

Comment:
@lwyszomi Is this correct? I don't see any instances of this elsewhere in the code, but I also don't see any other refs to the value before either.

For now assuming that these are views that have not yet been built